airbyte/destinations/util.py

```
@@ -19,6 +19,7 @@
 def get_destination(
     name: str,
     config: dict[str, Any] | None = None,
+    config_change_callback: Callable[[dict[str, Any], int], None] | None = None,
```

Ruff

Undefined name `Callable`

(F821)